src/app/landing/LandingPage.jsx

```diff
@@ -105,7 +105,7 @@ class LandingPage extends Component {
               <p className="text-lighter">
                 See the service details, go get what you need. We don’t keep your personal data, so your privacy is protected.
               </p>
-              <GetStartedButton onClick={this.getStarted} primary fluid />
+              <GetStartedButton onClick={this.toggleModal} primary fluid />
```

Comment:
Why does the getStarted method still exist? Is it still used anywhere?

Reply:
Fixed, tbh I had rushed this out without cleaning up code. I'll be more diligent in future

Reply:
No worries. :)